gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -357,6 +358,50 @@ def _modify_lb_resources_name(self, stack_template, provider_ptg,
                 stack_template[resources_key][resource][
                     properties_key]['vip']['name'] += ptg_name
 
+    def _generate_lbv2_member_template(self, is_template_aws_version,
+                                       member_ip, stack_template):
+        type_key = 'Type' if is_template_aws_version else 'type'
+        properties_key = ('Properties' if is_template_aws_version
+                          else 'properties')
+        resources_key = 'Resources' if is_template_aws_version else 'resources'
+        res_key = 'Ref' if is_template_aws_version else 'get_resource'
+
+        lbaas_loadbalancer_key = self._get_heat_resource_key(
+            stack_template[resources_key],
+            is_template_aws_version,
+            "OS::Neutron::LBaaS::LoadBalancer")
+        subnet = stack_template[resources_key][lbaas_loadbalancer_key][
+            properties_key]['vip_subnet']
+
+        return {type_key: "OS::Neutron::LBaaS::PoolMember",
+                properties_key: {
+                "pool": {res_key: "pool"},
+                "address": member_ip,
+                "protocol_port": {"get_param": "app_port"},
+                "subnet": subnet,
+                "weight": 1}}
+
+    def _modify_lbv2_resources_name(self, stack_template, provider_ptg,
+                                    is_template_aws_version):
+        # TODO(jiahao): Don't know why we have to change resources name
+        pass
```

Comment:
To distinguish between multiple lbaas. Add in similar manner as existing code is adding.